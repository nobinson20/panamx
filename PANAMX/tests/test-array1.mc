int main() {
    int i;
    int[5] a;
    double[3] b;
    a = [1024,22,33,-666,128];
    b = [-1.23, 5.56, 7.62];
    for (i = 0; i < 5; i++) {
        print(a[i]);
    }
    for (i = 0; i < 3; i++) {
        printf(b[i]);
    }
    return 0;
}
