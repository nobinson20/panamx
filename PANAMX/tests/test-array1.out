1024
22
33
666
128
